components/resources/speedreader_strings.grdp

```diff
@@ -1,31 +1,45 @@
-    <message name="IDS_READER_MODE_APPEARANCE_THEME_LIGHT" desc="Tooltip for light theme button">Light theme</message>
-    <message name="IDS_READER_MODE_APPEARANCE_THEME_SEPIA" desc="Tooltip for light theme button">Sepia theme</message>
-    <message name="IDS_READER_MODE_APPEARANCE_THEME_DARK" desc="Tooltip for light theme button">Dark theme</message>
-    <message name="IDS_READER_MODE_APPEARANCE_THEME_SYSTEM" desc="Tooltip for system theme button">System theme</message>    
-    <message name="IDS_READER_MODE_APPEARANCE_FONT_SANS"  desc="Tooltip for sans font button">Sans</message>
-    <message name="IDS_READER_MODE_APPEARANCE_FONT_SERIF"  desc="Tooltip for serif font button">Serif</message>
-    <message name="IDS_READER_MODE_APPEARANCE_FONT_MONO"  desc="Tooltip for mono font button">Monospace</message>
-    <message name="IDS_READER_MODE_APPEARANCE_FONT_DYSLEXIC"  desc="Tooltip for dyslexic font button">Dyslexic</message>
-    <message name="IDS_READER_MODE_APPEARANCE_FONT_SIZE_DECREASE"  desc="Tooltip for decrease font size button">Decrease the font size</message>
-    <message name="IDS_READER_MODE_APPEARANCE_FONT_SIZE_INCREASE"  desc="Tooltip for increase font size button">Increase the font size</message>
-    <message name="IDS_READER_MODE_TEXT_TO_SPEECH" desc="Tooltip for text-to-speech button">Text to speech</message>
-    <message name="IDS_READER_MODE_TEXT_TO_SPEECH_REWIND"  desc="Tooltip for rewind button">Rewind</message>
-    <message name="IDS_READER_MODE_TEXT_TO_SPEECH_PLAY_PAUSE"  desc="Tooltip for play/pause button">Play/Pause</message>
-    <message name="IDS_READER_MODE_TEXT_TO_SPEECH_FORWARD"  desc="Tooltip for forward button">Forward</message>
-    <message name="IDS_READER_MODE_TEXT_TO_SPEECH_SPEED_DECREASE"  desc="Tooltip for decrease playback speed button">Decrease the playback speed</message>
-    <message name="IDS_READER_MODE_TEXT_TO_SPEECH_SPEED_INCREASE"  desc="Tooltip for increase playback speed button">Increase the playback speed</message>
-    <message name="IDS_READER_MODE_SPEEDREADER" desc="Tooltip for disable/enable speedreader button">Speedreader mode toggle</message>
-    <message name="IDS_READER_MODE_AI" desc="Tooltop for AI button">Brave AI</message>
-  </if>
+    <message name="IDS_SPEEDREADER_SHOW_ORIGINAL_PAGE_LINK" desc="Text of the link to show original page">View original</message>
+    <message name="IDS_READER_MODE_MINUTES_TEXT" desc="Text of the read time element">min. read</message>
+
+    <if expr="not is_android">
+      <message name="IDS_SPEEDREADER_ICON_TURN_OFF_READER_MODE" desc="Accessible text shown when a page is in Reader Mode">Turn off speedreader</message>
+      <message name="IDS_SPEEDREADER_ICON_TURN_ON_READER_MODE" desc="Accessible text shown when a page can be distilled">Turn on speedreader</message>
+      <message name="IDS_READER_MODE_SHOW_ORIGINAL_PAGE_LINK" desc="Text of the link to show original page">View original</message>
```

Comment:
Is this used? Is it meant to replace `IDS_SPEEDREADER_SHOW_ORIGINAL_PAGE_LINK` (above, line 4)?

Reply:
No, it doesn't. removed.